### ISSUE TITLE
sensu-service fails to restart if stale PID-file is found

### ISSUE THREAD

**Issue by UnitedMarsupials** [sensu/sensu#1295]
If a sensu-service died (or was killed) for whatever reason -- and left a stale PID-file, the `sensu-service` script will fail to `restart` it:

```
# service sensu-client restart
Stopping sensu-client/etc/init.d/sensu-service: line 263: kill: (8553) - No such process
                                                           [FAILED]
```

Using `start` instead of `restart` works. There should be no difference -- the script may complain (and/or log) encountering a stale PID-file, but needs to proceed to starting the service anyway.

**Comment by calebhailey** [comment]
The 0.25 release is going to hit today with fewer features than originally anticipated & primarily only some internal improvements. Moving this issue to 0.26.

**Comment by vaibhavsoodibm** [comment]
Hi,

I am seeing this issue and see the "Fixed in Upcoming Release" tag, will this be fixed in the upcoming release (2.7)?

**Comment by cwjohnston** [comment]
Hi @vaibhavsoodibm, thanks for your inquiry.

I took another look at this today and found that thus far our changes for Sensu 0.27 had not yet addressed this condition. 

I believe changes in https://github.com/sensu/sensu-omnibus/issues/63 address this case. That change will ship with the 0.27 release.
